### ISSUE TITLE
drl_stop() results in difference between LED and robot state

### ISSUE THREAD

**Issue by Promation** [doosan-robotics/API-DRFL#21]
Hi,

We're using the API to control a v2.10.1 robot from an external PC.

When executing a drl_stop() command, the program stops correctly and the robot LEDs go to blue state.
However, the robot state number (that we're monitoring via profinet), often shows the wrong state.

E.g. when the robot was moving when a drl_stop() is executed, the robot state is 15 (standalone standby)
Or when a protective stop was active the robot state is 5 (interrupted).
Although the LEDs on the robot are blue.

**Comment by doosan-robotics** [comment]
@Promation 
Do you come across a situation where the LED and the actual robot status do not match? I will comment again after internal testing. Thank you for your report.

**Comment by Promation** [comment]
@doosan-robotics 
Yes, they do not match. The robot LED's are blue but the robot status is not number 11. Freedriving is also not possible then using the buttons on the cockpit.
